etcd/datadog_checks/etcd/data/conf.yaml.example

```
@@ -17,128 +20,147 @@ init_config:
     #   http: http://<PROXY_SERVER_FOR_HTTP>:<PORT>
     #   https: https://<PROXY_SERVER_FOR_HTTPS>:<PORT>
     #   no_proxy:
-    #     - <HOSTNAME_1>
-    #     - <HOSTNAME_2>
+    #   - <HOSTNAME_1>
+    #   - <HOSTNAME_2>
+
+    ## @param skip_proxy - boolean - optional - default: false
+    ## If set to `true`, this makes the check bypass any proxy
+    ## settings enabled and attempt to reach services directly.
+    #
+    # skip_proxy: false
+
+    ## @param timeout - number - optional - default: 10
+    ## The timeout for connecting to services.
+    #
+    # timeout: 10
+
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
 
+## Every instance is scheduled independent of the others.
+#
 instances:
 
-  -  
+  -
     ## @param use_preview - boolean - optional - default: true
-    ## Whether or not to preview the new version of the check supporting only ETCD 3+
+    ## Whether or not the preview to new version of the check supporting only ETCD 3+
     #
     # use_preview: true
 
     ## @param prometheus_url - string - required
     ## The URL where your application metrics are exposed by Prometheus.
     #
-    # prometheus_url: http://localhost:2379/metrics
+    prometheus_url: http://localhost:2379/metrics
 
-    ## @param ssl_cert - string - optional
-    ## The certificate to use when connecting to the `prometheus_url`.
-    ## It may also contain an unencrypted private key to use.
-    ##
-    ## NOTE: Use only if `use_preview` is set to `true`.
-    ##       This option is used in place of `tls_cert`
+    ## @param prometheus_metrics_prefix - string - optional
+    ## <PREFIX> for exposed Prometheus metrics.
     #
-    # ssl_cert: <CERT_PATH>
+    # prometheus_metrics_prefix: <PREFIX>_
```

```suggestion
    # prometheus_metrics_prefix: <PREFIX>
```
Not sure where this extra `_` came from

---

From templates, so it's actually more accurate than the old example 😅